### PR TITLE
Bring up

### PR DESCRIPTION
**Pull request by Lewis-Seiden**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/feeder/FeederIOReal.java [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
 
 /** Feeder IO using a TalonFX. */
 public class FeederIOReal implements FeederIO {
-  private final TalonFX motor = new TalonFX(13);
+  private final TalonFX motor = new TalonFX(13, "canivore");
 
-  DigitalInput firstBeambreak = new DigitalInput(0);
-  DigitalInput lastBeambreak = new DigitalInput(1);
+  InvertedDigitalInput firstBeambreak = new InvertedDigitalInput(0);
+  InvertedDigitalInput lastBeambreak = new InvertedDigitalInput(1);
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/feeder/FeederIOReal.java:20]: public or private?

File: src/main/java/frc/robot/subsystems/swerve/ModuleIOReal.java [hunk 2]
```diff
@@ -70,22 +71,24 @@ public class ModuleIOReal implements ModuleIO {
...
 
   public ModuleIOReal(ModuleConstants constants) {
     name = constants.prefix();
 
-    driveTalon = new TalonFX(constants.driveID());
-    turnTalon = new TalonFX(constants.turnID());
-    cancoder = new CANcoder(constants.cancoderID());
+    driveTalon = new TalonFX(constants.driveID(), "canivore");
+    turnTalon = new TalonFX(constants.turnID(), "canivore");
+    cancoder = new CANcoder(constants.cancoderID(), "canivore");
```
Comment (Awesomeplayer165) [inline src/main/java/frc/robot/subsystems/swerve/ModuleIOReal.java:82]: public or private?

**Review by Awesomeplayer165 — CHANGES_REQUESTED** [review]
small things.

File: src/main/java/frc/robot/subsystems/swerve/ModuleIOReal.java (hunk 2 above)
Comment (Lewis-Seiden) [reply src/main/java/frc/robot/subsystems/swerve/ModuleIOReal.java:82]: this isnt a declaration its just initialization

**Review by Awesomeplayer165 — APPROVED** [review]

**Pull request merged**